CMakeLists.txt

```diff
@@ -17,8 +17,7 @@ endif()
 set(FLAGS_WARNINGS "-Wall -Wextra -Werror -Wno-unused-variable -Wno-unused-value -Wno-deprecated-declarations")
 
 # In debug use the three simultaneously supported sanitizers, see https://stackoverflow.com/a/47261999
-set(CMAKE_CXX_FLAGS_DEBUG "-g -O0 ${FLAGS_WARNINGS} -fsanitize=address -fsanitize=undefined -fsanitize=leak")
```

Comment:
why no more sanitize?

Reply:
It's now in the Dockerfile arguments

Reply:
`docker/ci-sanitize/Dockerfile` as an ENV